pydatalab/pydatalab/models/people.py

```diff
@@ -91,6 +92,21 @@ def add_missing_type(cls, v):
     def set_default_type(cls, _):
         return "people"
 
+    @validator("display_name")
+    def validate_display_name_length(cls, v):
+        """Validate that the display name."""
+        if len(v) > 150:
+            raise ValueError("Display name must be at most 150 characters long.")
+        return v
```

Comment:
Another approach (related to other comment) would be to define a custom type called `DisplayName` (or something) that is just a constrained string less than 150 chars long. We can then directly validate strings with it, without having to create the entire `Person` model